src/openrct2/ride/gentle/CarRide.cpp

```diff
@@ -433,16 +433,16 @@ static void PaintCarRideTrackLeftQuarterTurn1Tile(
     switch (direction)
     {
         case 0:
-            PaintAddImageAsParent(session, imageId, { 6, 0, height }, { 26, 24, 1 }, { 6, 2, height });
+            PaintAddImageAsParent(session, imageId, { 6, 0, height }, { { 6, 2, height }, { 26, 24, 1 } });
             break;
         case 1:
-            PaintAddImageAsParent(session, imageId, { 0, 0, height }, { 26, 26, 1 });
+            PaintAddImageAsParent(session, imageId, { 0, 0, height }, { { 0, 0, height }, { 26, 26, 1 } });
             break;
         case 2:
-            PaintAddImageAsParent(session, imageId, { 0, 6, height }, { 24, 26, 1 }, { 2, 6, height });
+            PaintAddImageAsParent(session, imageId, { 0, 6, height }, { { 2, 6, height }, { 24, 26, 1 } });
             break;
         case 3:
-            PaintAddImageAsParent(session, imageId, { 6, 6, height }, { 24, 24, 1 });
+            PaintAddImageAsParent(session, imageId, { 6, 6, height }, { { 0, 0, height }, { 24, 24, 1 } });
```

Comment:
```suggestion
            PaintAddImageAsParent(session, imageId, { 6, 6, height }, { { 6, 6, height }, { 24, 24, 1 } });
```

Again, please make sure all the occurrences are correct, this has such a high potential of introducing errors.

Reply:
Perhaps we should add unit tests for those painting methods to make sure we don't break anything...